### PR TITLE
Add organization command

### PR DESCRIPTION
**Pull request by Ladicle**

### CODE REVIEW

File: internal/auth/org/org.go [hunk 1]
```diff
@@ -0,0 +1,218 @@
+package org
```
Comment (kkaneda) [inline internal/auth/org/org.go:1]: Just a comment. I was wondering if `orgs` is better, but I myself has not been consistent on this..

File: internal/auth/org/org.go [hunk 2]
```diff
@@ -0,0 +1,218 @@
...
+	}
+	cmd.Flags().StringVar(&title, "title", "", "Title of the organization")
+	_ = cmd.MarkFlagRequired("title")
+	return cmd
+}
+
+func addMemberCmd() *cobra.Command {
+	var title, email, role string
+	cmd := &cobra.Command{
+		Use:  "add-member",
```
Comment (kkaneda) [inline internal/auth/org/org.go:79]: Maybe `add-user`? No strong opinion, though.

**Review by kkaneda — APPROVED** [review]
LG!

File: internal/auth/org/org.go (hunk 2 above)
Comment (kenji-cloudnatix) [reply internal/auth/org/org.go:79]: I guess `add-member` is better when we want to add a list command (`list-users` v.s. `list-members`).

**Comment by kkaneda** [comment]
Let me go ahead and merge this. Thanks!

**Pull request merged**